clientapi/clientapi_service.py

```diff
@@ -141,16 +140,9 @@ def ping_client(self, request):
             return BaseResponse(code=404, message="Invalid push token for user")
         else:
             client = clients[0]
-            response = NotificationHelper.send_ping(client)
-            # If we got a response from the send_ping method, it was sent via TBANS
-            # We'll bubble up any errors we got back
-            if response:
-                if response.code == 200:
-                    return BaseResponse(code=200, message="Ping sent")
-                else:
-                    return BaseResponse(code=response.code, message="Error pinging client - {}".format(response.message))
-            else:
-                return BaseResponse(code=200, message="Ping sent")
+            from helpers.tbans_helper import TBANSHelper
+            TBANSHelper.ping(client)
```

Comment:
- [ ] We should be using the response of these calls to send back a success or error to clients